### DISCUSSION TITLE
Enable versioning for nested entries

### DISCUSSION THREAD

**Discussion by jamiepittock** [craftcms/cms#15715] [Ideas]
Sections have this setting:

![Screenshot 2024-09-11 at 08 55 45](https://github.com/user-attachments/assets/e3e2c8ad-9ec1-4777-81bb-2df221550baf)

Can the setting be added to Matrix fields, so that nested entries can be versioned? At least when the view mode is "as cards" or "as an element index".

At the moment, even though these nested entries can be edited in the same way as regular/non-nested entries, they can't be versioned.

![Screenshot 2024-09-11 at 09 00 57](https://github.com/user-attachments/assets/99bc1ce6-f93b-475b-9d07-d2f29ac43e3c)

**Comment by thupsi** [comment]
I second this. Lack of versioning can be a blocker if one plans to use a Matrix field as a nested section.
